### DISCUSSION TITLE
FR: Read-only access

### DISCUSSION THREAD

**Discussion by kodek** [coddingtonbear/obsidian-local-rest-api#119] [Ideas]
Hi there,

I'm so excited to learn about this extension. Any chance to add a new bearer token for read-only access? I'd love to experiment with the API, but I'm worried I might shoot myself in the foot with it :)

Thanks!

**Comment by coddingtonbear** [comment]
I'm not personally up for adding support for more than one API key (and API key type), but you might find it useful to know that you can avoid making any changes at all to your notes if you just stick to `GET`-type requests.  `REST` semantics don't allow `GET` requests to make changes to anything, and I believe all of the other request methods definitely *do* make changes.
